### PR TITLE
Removes brig phys, adds another officer slot to compensate

### PR DESCRIPTION
**Pull request by AffectedArc07**
## What Does This PR Do
Removes brig physician and adds another officer slot (and equipment) to compensate. 

## Why It's Good For The Game
This role has long been eroded from what it originally was. It was originally a role put in by ponies to make the karma shop more appealing, and had the goal of healing prisoners who got harmed in arrests.

This never happened, the role just turned into a private medbay for security to get their own implants and other stuff with zero medbay risk.

"B-but the brig needs someone to help out in"
1. We have a warden
2. You arguing this point just further proves how this role has been eroded from its original purpose, and how its just gotten worse and worse over time

This is a continuation from the meeting, and this seems to be the way forward.

A brig assistant/associate/helper role to assist the warden may come in the future, but this is happening now

## Images of changes
See MapDiffBot

## Changelog
:cl: AffectedArc07
del: Removes Brig Physician. You can refund it for 5 karma points if you bought it.
add: Added another sec officer slot 
/:cl:

### CODE REVIEW

**Review by S34NW — CHANGES_REQUESTED** [review]
Delta:
image

Meta:
image


Please map this slightly more sanely

**Comment by Vangar200** [comment]
As someone who actually plays the roll often, it makes perfect sense to leave it in. It's a trusted member of medical (because of their implant) who's not going to steal anything from the officers or hurt any of the prisoners due to the officers not knowing how to use force sometimes. It also doesn't make sense for a warden to be practicing medicine. Yes it's mostly small things, but it still isn't his job. 

He's basically the blueshield but for security, if you're going to argue to get rid of him then might as well get rid of the blueshield/change his job description. If players are missplaying a role you don't get rid of it. You make guidelines for the role and have the warden do his job and run his/her prison correctly.

**Review by S34NW — APPROVED** [review]
mapping is now fine

**Comment by AffectedArc07** [comment]
> As someone who actually plays the roll often, it makes perfect sense to leave it in.

The amount of time *one* person has spent in a role doesn't change the fact its eroded far from its original purpose into something much much worse.

---

> It's a trusted member of medical (because of their implant) who's not going to steal anything from the officers

This is one of the reasons its bad. Officers should run the same risk as anyone using medbay. Having officers need to use the normal medbay opens up more interesting dynamics and other things, opportunities for deal making and other traitorous RP acts

---

> or hurt any of the prisoners due to the officers not knowing how to use force sometimes. 

The amount of times this happens is minimal, and in speaking to Necaladun, the general concensus was this
```ini
[AffectedArc07]
I assume prisoners with minor scuffs can be told to just get a grip
and if its major they can have a trip to medbay

[Necaladun]
oh 100%
or they can just wait?
if its not lifethreatening
suck it up
```

---

> It also doesn't make sense for a warden to be practicing medicine. Yes it's mostly small things, but it still isn't his job.

I never said the warden should be practicing medicine. No one in the brig should be. Thats what medbay is for.

---

> He's basically the blueshield but for security, if you're going to argue to get rid of him then might as well get rid of the blueshield/change his job description. If players are missplaying a role you don't get rid of it. You make guidelines for the role and have the warden do his job and run his/her prison correctly.

The brig physician isn't the security blueshield **at all**. The blueshield is a bodyguard and protector for command. You saying this right here is concrete proof that the roll has eroded far from its original intentions into something much much worse. Security officers shouldnt have a combat medic on hand as a bodyguard. Just no.

**Comment by hal9000PR** [comment]
`He's basically the blueshield but for security` why does security need a blueshield?

`blueshield/change his job description.` blueshields don't normally make medbay in the bridge

`You make guidelines for the role and have the warden do his job and run his/her prison correctly.` SOP changes don't do much to influence player behaviour. miners cannot give out mining loot as of SOP, they still do. cos nobody gives a shit.

`It's a trusted member of medical (because of their implant)` exactly the issue with brig phys. they can do all of medbays job without any of the risk. it cuts down on the number of interactions a player has. less interactions, the less interesting the outcome of the round is. it might be useful for sec to have a "heal zone" but its not good game design. all of medbay should be in medbay. imagine if the brig had a "brig engineer" for when perma gets hit by asteroids. it would be dumb right? why make an exception for brig phys. brig phys goes against one of the core concepts of SS13. departmental co-operation. they go against this fundamental design concept so hard that a sec officer often doesn't have to consider medbays existance. an entire department is kinda just nulled by 1 job.

**Comment by Vangar200** [comment]
But I'm not the only person who plays that roll correctly either, I'm just currently the only one saying anything. And have either of you actually been to a prison? A base? Anything that has a secure area? They have their own medical treatment facility there for minor things. It reduces the risk of many different situations and makes sense to have. At this point it seems like you're fighting to get rid of it to allow antags to have a small buff. 

Also I was literally dead and sitting in a chair getting interrogated in security. 9 times out of 10 they don't care if youre hurt. They won't take you to medbay. Even after I was dead they put my in a cell for a few minutes before they thought I committed suicide.

I meant the blueshield part as in how he's ensures the command staff is healthy, I know for a fact the Brig Doc isn't supposed to be anyone's bodyguard.

**Comment by hal9000PR** [comment]
`They have their own medical treatment facility there` sec has this. its called medbay

**Comment by AffectedArc07** [comment]
> But I'm not the only person who plays that roll correctly either, I'm just currently the only one saying anything.

You dont play it correctly though, you have literally mentioned how you think its a "blueshield for security"

---

> And have either of you actually been to a prison? 

Oddly enough, no. I have not been to prison.

---

> A base? Anything that has a secure area?  They have their own medical treatment facility there for minor things. 

Yes, I have been to an RAF campus numerous times. They did not have an on-site miniature hospital, they had first aid kits in places like pretty much everywhere else does.

---

> It reduces the risk of many different situations and makes sense to have. At this point it seems like you're fighting to get rid of it to allow antags to have a small buff.

If I wanted to buff antags I would do that straight up. I have been developing here since 2017, Maintained this codebase for a good year, and currently host the server. I am aware that stealth buffing things doesnt apply, and this seems like you are clutching at *anything* to make a case here

---

> Also I was literally dead and sitting in a chair getting interrogated in security. 9 times out of 10 they don't care if youre hurt. They won't take you to medbay. Even after I was dead they put my in a cell for a few minutes before they thought I committed suicide.

If security is interrogating a dead person, we have bigger problems than "oh no my brig medbay"

---

> I meant the blueshield part as in how he's ensures the command staff is healthy, I know for a fact the Brig Doc isn't supposed to be anyone's bodyguard.

Again. You are literally proving my point that this role has eroded far from its original purpose. **They. Are. Not. A. Private. Doctor. For. Security.**

**Comment by Vangar200** [comment]
> 

Literally said how I meant the medical portion of blueshield to keep them healthy. Also if you look at the guide for it, it even states "a Brig Physician's only job is to treat officers and prisoners"   

The point for me bringing how I was dead is I literally died in the chair. Could I have been saved? Yes. but there was no Brig Doctor there to do anything because nobody had the role yet. 

And you prove my point of brig medical being a first aid station. Patching up what they can and if it's more than they, they go to medical.

**Comment by Gatchapod** [comment]
> > 
> 
> Literally said how I meant the medical portion of blueshield to keep them healthy. Also if you look at the guide for it, it even states "a Brig Physician's only job is to treat officers and prisoners"

Yes, it does. It did even back in 2014 when the role was apparently added. And that changes nothing.
After all these years we can clearly see adding it was a mistake.

MDs can roll antags. Everyone is susceptible to antag MDs, with exception of security. Why?
Seriously, give one strong argument why security should get a special treatment.

Otherwise, BP exists solely to deny antags opportunities and that's completely anti-fun.

**Comment by Vangar200** [comment]
> > Literally said how I meant the medical portion of blueshield to keep them healthy. Also if you look at the guide for it, it even states "a Brig Physician's only job is to treat officers and prisoners"
> 
> Yes, it does. It did even back in 2014 when the role was apparently added. And that changes nothing. After all these years we can clearly see adding it was a mistake.
> 
> MDs can roll antags. Everyone is susceptible to antag MDs, with exception of security. Why? Seriously, give one strong argument why security should get a special treatment.
> 
> Otherwise, BP exists solely to deny antags opportunities and that's completely anti-fun.

Because when you add a bigger chance of security officers to get killed/looted, it just means antags can roam more freely. Nobody likes walking into the station every other round half blown up and half with bodies everywhere because security was already taken out. Nobody can do anything harmful to antags except security. Yes you can a minor amount but you can’t go out of your way to stop them. Even then it’s not like the BP is  god and fixing up broken bones. 

It’s such a minor role it’s more of a convenience than anything, so there’s no point to taking it out. You’re making it seem like it’s only in the way of everything. 

Does BP stop them from dying? No. What’s to stop a borg or someone building a BP room every round and actually making it more effective than it is now? Not much. I still don’t see why we’d take it out. There’s not been a single reason except “another officer slot and changing how security looks a little”

**Comment by matttheficus** [comment]
>Because when you add a bigger chance of security officers to get killed/looted, it just means antags can roam more freely.

Medbay is already valid-central. If you even THINK about closing the shutters some curious person is gonna kick the door down and accuse you of being a Shadowling-Nukie-Cultist.

>Even then it’s not like the BP is god and fixing up broken bones.

This is nearly every BP ever. That and implant checks right in the safety of the Brig.

>It’s such a minor role it’s more of a convenience than anything, so there’s no point to taking it out.

It defeats a core tenant of Antag-Security design that I dont believe you understand, as its been explained a few times above already.

>What’s to stop a borg or someone building a BP room every round and actually making it more effective than it is now? Not much.

This would fall under powergaming, just as Cargo making RND every round.

>There’s not been a single reason except “another officer slot and changing how security looks a little”

If this is your honest take, you are letting your extreme favor toward BP blind you from anything anyone above has said at all.

I get that losing your favorite role is a pain. I personally only really played SPP and Warden back when Pods were still a thing but I understand that proper code (in the SPP's case) and proper game design (in the BP's case) are more important for the long-term health of the server. If you are going to argue against its removal, ignoring the absolutely sane points that both long-term contributors, players, and Security veterans have stated above you are simply trying to bully your opinion onto everyone else.

**Comment by Necaladun** [comment]
Overall, necessary for long-term health. Security should be relying on medbay for healing, not having a role dedicated to bypassing it and certainly not having their own mini-medbay. 

A medic is a cool idea, but a 'security engineer' could also be cool and fun. Neither is good for overall balance, so this gets the old stamp-of-approval.

**Comment by Rurik123** [comment]
Officer as compensation doesn't really make much sense when sec isn't losing field presence, just the ability to slap a patch on a prisoner. Better off axing it without adding another officer. Or, if you have to add counter weight in some way, just a basic first aid kit or two in security. (Not the menders, just the patches)

**Comment by Carthusia** [comment]
After some thought, even though i have enjoyed alot of my time as a brig phys, i can admit that i probably havent been playing it with its original indended purpose, and with that purpose, its entire function can be relegated to having the wardens locker spawn with 1 regular first aid kit and 1 advanced first aid kit.

The warden is meant to be handling detained suspects anyways and if Bob comes in with some bruises, scratches and a cigarette burn or two he can just grab the kit.

**Comment by Coolrune206** [comment]
Would it be possible to give the brigs an Emergency Nanomed to give them some means to treat minor bruises and burns on prisoners without having to yoink other public supplies/medbay's meds to do so?

**Comment by AffectedArc07** [comment]
> Officer as compensation doesn't really make much sense when sec isn't losing field presence, just the ability to slap a patch on a prisoner. Better off axing it without adding another officer. Or, if you have to add counter weight in some way, just a basic first aid kit or two in security. (Not the menders, just the patches)

This is so we're not losing more and more job slots.

**Comment by Rurik123** [comment]
> > Officer as compensation doesn't really make much sense when sec isn't losing field presence, just the ability to slap a patch on a prisoner. Better off axing it without adding another officer. Or, if you have to add counter weight in some way, just a basic first aid kit or two in security. (Not the menders, just the patches)
> 
> This is so we're not losing more and more job slots.

Ah, roger. Fair enough

**Comment by roarbark** [comment]
there's good reasoning behind removing bp, but, from my perspective, there's one thing people have been overlooking. hurt officers are gonna have their patrol partner glued to the OR windows like they do currently when theres no bp and instances where too much of security is dead for that to happen i dont think its healthy for the round if seccies get game ended in ORs instead of being revived.

**Comment by roarbark** [comment]
> Would it be possible to give the brigs an Emergency Nanomed to give them some means to treat minor bruises and burns on prisoners without having to yoink other public supplies/medbay's meds to do so?

i'm pretty sure cyberiad has one infront of the armoury, im unsure about the other maps

**Comment by RagingLemons1** [comment]
I don't like this. I've had fun playing as Brig phys and don't want to see it removed.

File: code/game/jobs/jobs.dm [hunk 1]
```diff
@@ -93,7 +92,6 @@ GLOBAL_LIST_INIT(whitelisted_positions, list(
 	"Blueshield",
 	"Nanotrasen Representative",
 	"Barber",
```
Comment (Sirryan2002) [inline code/game/jobs/jobs.dm:94]:
```suggestion
	"Barber"
```
I believe we need to drop the comma here yes?

File: code/modules/karma/karma_holder.dm [hunk 2]
```diff
@@ -390,8 +391,6 @@
 					switch(href_list["KarmaBuy"])
 						if("1")
 							karma_purchase(5, "job", "Barber")
-						if("2")
```
Comment (Sirryan2002) [inline code/modules/karma/karma_holder.dm:393]: this wont cause any problems with the option values being 1, 3, 4 right?

**Comment by hal9000PR** [comment]
> I don't like this. I've had fun playing as Brig phys and don't want to see it removed. 

Just because something is fun doesn't mean its healthy for the game. running around with meth + nuka cola at light speed was "fun" but its not healthy for the overall gameplay. 

Some things are fun for the user and nobody else. brig phys is a net negative.

File: code/modules/karma/karma_holder.dm (hunk 2 above)
Comment (AffectedArc07) [reply code/modules/karma/karma_holder.dm:393]: Nope

File: code/game/jobs/jobs.dm (hunk 1 above)
Comment (AffectedArc07) [reply code/game/jobs/jobs.dm:94]: Its a styling thing, but DM lists still compile with trailing commas

**Comment by LiberatedWaveMan** [comment]
I've seen some good brig physicians in my time, but most of them just make it a private security medbay, give out combat implants, and completely mitigate the need for sec officers to enter medbay at any point in the round. It will not be missed.

**Comment by Denthamos** [comment]
Security should not be cutting out the actual medical roles from the equation with the brig doctor and preventing department interaction in that manner and unfortunately over the years the brig doctor has evolved into a one stop shop for security to get healed, get implants, sometimes even get cloned with backups of them being made there.  It is not healthy for the state of the game for the role to continue on as it is.

**Comment by Spacemanspark** [comment]
Just going to peek out of my hidey hole to say that this is one of the few karma roles I was ever okay with removing. Security doesn't need a private medic for combat implants, and I got rather sick of brig physicians pestering me for them constantly as a roboticist. It was also rather obnoxious to see them setting up god-tier medical facilities in the small space they were given and becoming self sufficient without any downsides.

**Comment by LordShekelstein** [comment]
I'm glad to see some of the festering wounds gradually being excised from Paradise; swarmers getting the axe is also just incredible. The peanut gallery will likely decry from the back of the hall, "improve don't remove," however, with brig physician, was there really anything to improve here? The actual role was fine, no doubt -- no code issues, no general gameplay issues, and as many have cited in this very discussion thread, the role was indeed, fun to play -- however the culture which has formed around it became too far removed from the initial intent leaving it in unhealthy for the game. SS13 is a pretty expansive sandbox, so it's unlikely you'll balance everything correctly, impossible, even. But I believe this is a good step forwards, nice work, Affected.

**Comment by EmilyFontaine** [comment]
Bye, Felicia.

**Review by dearmochi — APPROVED** [review]

**Pull request merged**